### ISSUE TITLE
A11y_VSCode_SidebarBarOptions_AccessingTheDebugsidebaroptions_ScreenReader: Identical names provided for the “Collapse all” buttons for “Variables” and “Watch” toolbar.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode#76986]
**Environment Details:**
VSCode Version : 1.35.6 (user setup)
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 4.1.2
ScreenReader Versions : NVDA2019.1.1, JAWS 2019

**Repro Steps:**
1. Open Visual studio code editor.
2. Using "Ctrl+Shift+D" key navigate to the debug side view bar option.
3. Using "Tab" key navigate to Debug side bar options.
4. Start screen reader.
5. Navigate to the “Collapse all” buttons for “Variables” and “Watch” toolbar.
6. Listen to how it is announced.

**Actual:**
If the more than one interactive element is provided with the same name for the controls in two different toolbars then the screen reader users will find it hard in knowing which control is for which section.

**Expected:**
The name provided for each of the controls should be unique so the screen reader users can differentiate the two controls from the two different toolbar.

**Recommendations:**
The “Collapse all” button in “Variables” toolbar should be announced as “Collapse all Variables” and for “Collapse all” button in “Watch” toolbar as “Collapse all Watch”.

Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
If the interactive elements are not provided with unique different names then the screen reader users might not be able to differentiate between them.

**MAS Reference:**
MAS 4.1.2 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Attachment for Reference:**
![VSCode_SidebarBarOptions_AccessingTheExtensionbaroptions_ScreenReader Identical names for “Collapse all” buttons](https://user-images.githubusercontent.com/52036772/60891288-a6396580-a27a-11e9-8341-b9c2b7c218f2.png)

**Comment by isidorn** [comment]
Dup of https://github.com/microsoft/vscode/issues/77054
